### ISSUE TITLE
Security Audit - Outdated dependencies

### ISSUE THREAD

**Issue by embano1** [cloudevents/spec#1132]
After reviewing the recent security [audit](https://github.com/cloudevents/spec/blob/main/docs/CE-SecurityAudit-2022-10.pdf) I was wondering whether we should enable Github Dependabot for this repo to automatically bump deps.

cc/ @duglin @lionelvillard

**Comment by duglin** [comment]
yep - just need to find the time :-)

**Comment by YohanSciubukgian** [comment]
As CloudEvents SDKs provide out of the box integration with 3rd party libraries, could we add either dependabot or renovate for managing all dependencies for all CloudEvents repositories?

For example, on the JAVA-SDK repository, the latest SDK update is from May 15, 2023 and the following packages have known vulnerabilities on 3rd party dependencies:
- https://mvnrepository.com/artifact/io.cloudevents/cloudevents-protobuf/2.5.0
- https://mvnrepository.com/artifact/io.cloudevents/cloudevents-json-jackson/2.5.0
- https://mvnrepository.com/artifact/io.cloudevents/cloudevents-kafka/2.5.0
- https://mvnrepository.com/artifact/io.cloudevents/cloudevents-spring/2.5.0
- etc.

**Comment by embano1** [comment]
Yes, we use Dependabot in the `sdk-go` [repo](https://github.com/cloudevents/sdk-go/blob/main/.github/dependabot.yml). Want to file a PR? Not sure how much work is involved though to integrate with Maven (security keys to push).
